<launch>

    <arg name="cam_source" default="cam1" />
    <arg name="cam_target" default="cam2" />
    <arg name="merged_name" default="scene" />

    <node pkg="calib" name="preproc_$(arg cam_source)" type="preprocessing_node" respawn="true" output="screen" args="/$(arg cam_source)/qhd/points $(arg cam_source)" />
    <node pkg="calib" name="preproc_$(arg cam_target)" type="preprocessing_node" respawn="true" output="screen" args="/$(arg cam_target)/qhd/points $(arg cam_target)" />

    <!--<node pkg="pcl_ros" name="target_model_publisher" type="pcd_to_pointcloud" respawn="true" output="screen" args="$(find calib)/models/scene_sim_preproc_20.pcd 0.5 _frame_id:=world" />
    <node pkg="calib" name="icp_$(arg cam_source)" type="icp_node" respawn="true" output="screen" args="$(arg cam_source) cloud_pcd" />
    <node pkg="calib" name="icp_$(arg cam_target)" type="icp_node" respawn="true" output="screen" args="$(arg cam_target) cloud_pcd" />-->
    
    <node pkg="calib" name="icp_$(arg cam_source)" type="icp_node" respawn="true" output="screen" args="$(arg cam_source) /calib/clouds/cam2/preproc" />

</launch>
